test/integration/utils/operator.go

```diff
@@ -11,10 +11,10 @@ import (
 	"github.com/shipwright-io/build/pkg/controller"
 )
 
-// StartBuildOperator initialize an operator as if being call from main,
+// StartBuildController initialize an operator as if being call from main,
 // but it disables the prometheus metrics and leader election. This intended
 // to for testing.
-func (t *TestBuild) StartBuildOperator() (chan struct{}, error) {
+func (t *TestBuild) StartBuildController() (chan struct{}, error) {
```

Comment:
I think I did a similar comment somewhere else, this function actually starts a manager, which is an interface in the controller-runtime pkg. I think `StartManager` is a better name, rather than a search/replace approach.

Reply:
Thanks!, two cents:
- I think the logic should be: start a controller and controller starts a manager, so manager should be included in the controller, and the goal of this file is used to start a controller for test, not start a manager as a internal usage.
- I am not sure if we want to add more config beside of manager in future to enrich the test.

So I prefer to keep StartBuildController. 

But I am fine for both names.

Reply:
Technically speaking this function does the following:
- Add all controllers to the manager, so we add four
- Starts all registered Controllers and blocks until the Stop channel is closed

For me `StartBuildController` is missleading, can we call this `StartControllers`?

Reply:
Agree and fixed. Thanks all